core/src/test/java/org/elasticsearch/search/aggregations/metrics/HDRPercentilesIT.java

```diff
@@ -57,13 +59,66 @@
 import static org.hamcrest.Matchers.notNullValue;
 import static org.hamcrest.Matchers.sameInstance;
 
-/**
- *
- */
-public class HDRPercentilesTests extends AbstractNumericTestCase {
+public class HDRPercentilesIT extends AbstractNumericTestCase {
+
     @Override
     protected Collection<Class<? extends Plugin>> nodePlugins() {
-        return Collections.singleton(GroovyPlugin.class);
+        return Collections.singleton(CustomScriptPlugin.class);
+    }
+
+    public static class CustomScriptPlugin extends MockScriptPlugin {
```

Comment:
Maybe this should be a top level class with javadoc rather than copy and paste? I'd try and keep it as close to the classes as possible, probably in the `org.elasticsearch.search.aggregations.` package? Maybe rename it to `AggregationTestScriptsPlugin` or something.